packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.tsx

```diff
@@ -0,0 +1,159 @@
+import { routes } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import CFDCompareAccountsCard from './cfd-compare-accounts-card';
+import {
+    getSortedCFDAvailableAccounts,
+    getEUAvailableAccounts,
+    getMT5DemoData,
+    getDxtradeDemoData,
+    dxtrade_data,
+} from '../../Helpers/compare-accounts-config';
+
+const CompareCFDs = observer(() => {
+    const history = useHistory();
+    const store = useStore();
+    const { client, traders_hub } = store;
+    const { trading_platform_available_accounts } = client;
+    const { is_demo, is_eu_user, available_dxtrade_accounts } = traders_hub;
+
+    const sorted_available_accounts = !is_eu_user
+        ? getSortedCFDAvailableAccounts(trading_platform_available_accounts)
+        : getEUAvailableAccounts(trading_platform_available_accounts);
+
+    // Check if dxtrade data is available
+    const has_dxtrade_account_available = available_dxtrade_accounts.length > 0;
+
+    const sorted_cfd_available_eu_accounts =
+        is_eu_user && sorted_available_accounts.length ? [...sorted_available_accounts] : [];
+
+    // Getting real accounts data
+    const all_real_sorted_cfd_available_accounts = !is_eu_user
+        ? [...sorted_available_accounts]
+        : [...sorted_cfd_available_eu_accounts];
+
+    // Getting demo accounts data
+    const demo_cfd_available_accounts = [
+        ...getMT5DemoData(all_real_sorted_cfd_available_accounts).concat(
+            getDxtradeDemoData(all_real_sorted_cfd_available_accounts)
+        ),
+    ];
+
+    const all_cfd_available_accounts =
+        is_demo && demo_cfd_available_accounts.length > 0
+            ? demo_cfd_available_accounts
+            : all_real_sorted_cfd_available_accounts;
+
+    // Calculate the card count for alignment of card in center
+    const card_count = has_dxtrade_account_available
+        ? all_cfd_available_accounts.length + 1
+        : all_cfd_available_accounts.length;
+
+    const DesktopHeader = (
+        <div className='compare-cfd-header'>
+            <div
+                className='compare-cfd-header-navigation'
+                onClick={() => {
+                    history.push(routes.traders_hub);
+                }}
+            >
+                <Icon icon='IcArrowLeftBold' />
+                <Text size='xs' weight='bold' color='prominent'>
+                    <Localize i18n_default_text="Trader's hub" />
+                </Text>
+            </div>
+            <h1 className='compare-cfd-header-title'>
+                <Text size='m' weight='bold' color='prominent'>
+                    <Localize
+                        i18n_default_text='Compare CFDs {{demo_title}} accounts'
+                        values={{
+                            demo_title: is_demo ? localize('demo') : '',
+                        }}
```

Comment:
It might have two spaces if is_demo is false

Reply:
Nope i already checked in real(is_demo=false) it is "Compare CFDs accounts"
<img width="1314" alt="Screenshot 2023-07-07 at 4 41 22 PM" src="https://github.com/binary-com/deriv-app/assets/120543468/9fe2fe32-3590-426e-bd67-5494d8817e38">
it is